bbreaker/GameObject.py

```diff
@@ -87,34 +113,20 @@ def handle_events(self):
                 self.handle_mouse_events(event)
 
     def handle_mouse_events(self, event):
-        if event.type == pygame.MOUSEBUTTONDOWN:
-            pygame.mouse.set_cursor(*pygame.cursors.broken_x)
-            mouseButtons = pygame.mouse.get_pressed()
-            if mouseButtons[0] == 1:
-                self.start_pos = event.pos
-
-        if event.type == pygame.MOUSEBUTTONUP:
-            pygame.mouse.set_cursor(*pygame.cursors.arrow)
-            if self.start_pos and pygame.mouse.get_pressed()[0] == 0:
-                self.end_pos = event.pos
-                self.strike()
-
-        if event.type == pygame.MOUSEMOTION:
-            if pygame.mouse.get_pressed()[0] == 1:
-                # pressed - draw line shot direction
-                pass
+        self.current_mpos = pygame.mouse.get_pos()
```

Comment:
Please, revert back striking logic. 
It would be too easy to point a specific brick and have therefore a perfect shot during all rounds. I think that showing a line of shot should be available only during first < 20 levels to make higher levels more difficult. With the line of shot a user will instantly learn how to use 'bow/arrow' logic.
At some point we can also add a bonus in higher levels (by destroying bonus brick / unlocking some kind of achievement) so that the player could decide to use a line of shot during particularly tough round (once per bonus).